src/app/services/organization.service.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { allOrganization, Organization } from '../models/organizations';
+import { Observable } from 'rxjs';
+import { environment } from '../../environments/environment';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class OrganizationService {
+  baseUrlOrg=environment.apiEndpointOrganization
+  constructor(private http:HttpClient) { }
+  getAllOrganizationsApi():Observable<allOrganization>{
+    // here extract data array and return 
+    const observableData =this.http.get<allOrganization>(this.baseUrlOrg+'/getall')
```

Comment:
build the string in a const variable and pass it to the api call

Reply:
built the string out of method getAllOrganizationsApi and then passed it to api call